scripts/generateProptypes.ts

```diff
@@ -180,6 +180,7 @@ async function generateProptypes(
       if (
         name.toLowerCase().endsWith('classes') ||
         name === 'theme' ||
+        name === 'ownerState' ||
```

Comment:
When I tested to generated the prop-types of the `styled`, it crashed because it was not able to parse `ownerState` (due to a property equal to undefined).
I think that it make sense to just not resolve this prop, like on the doc demos.